src/pages/CommandBlocklistView/CommandBlocklistView.tsx

```diff
@@ -0,0 +1,200 @@
+import AddIcon from '@mui/icons-material/Add'
+import DeleteIcon from '@mui/icons-material/Delete'
+import { Button, Checkbox, IconButton, Tooltip } from '@mui/material'
+import Divider from 'components/divider'
+import { ModalWrapper } from 'components/ModalWrapper'
+import PageHeader from 'components/PageHeader'
+import { Table } from 'components/Table'
+import { useBlockList } from 'hooks/useBlockList'
+import { generateCommandName, useCommands } from 'hooks/useCommands'
+import { useMemo, useState } from 'react'
+import { Column } from 'react-table'
+import { BlockedCommand, CommandRow } from 'types/custom_types'
+
+const ModalColumns = () => {
```

Comment:
`ModalColumns` and `TableColumns` should not be capitalized (see discussion below for line 78). They can be renamed to `useModalColumns` and `useTableColumns` because they are, in fact, hooks.

Personally, I would prefer these to be in their own file so as to shorten the file, thereby reducing the cognitive load when reading this file. That's the pattern used (before this PR) in `CommandIndexTable`, for example (though I'm first to admit `data.tsx` was not the best choice of names). We can discuss if you thinks that's a bad idea.